bundle/edu.gemini.spModel.io/src/test/scala/edu/gemini/spModel/io/impl/migration/to2020A/SchedulingBlockTest.scala

```diff
@@ -2,17 +2,29 @@ package edu.gemini.spModel.io.impl.migration.to2020A
 
 import edu.gemini.pot.sp.{ISPObservation, ISPProgram}
 import edu.gemini.shared.util.immutable.ScalaConverters._
+import edu.gemini.spModel.core.{Semester, Site}
 import edu.gemini.spModel.io.impl.migration.MigrationTest
 import edu.gemini.spModel.obs.{SchedulingBlock, SPObservation}
 import edu.gemini.spModel.rich.pot.sp._
 import org.specs2.mutable.Specification
 
+import scala.math.Ordering.Implicits._
+
 
 class SchedulingBlockTest extends Specification with MigrationTest {
 
   val ProgramName: String =
     "GS-2020A-Q-501.xml"
 
+  val sem2020A: Semester =
+    new Semester(2020, Semester.Half.A)
+
+  def currentSemester: Semester =
+    new Semester(Site.GS, System.currentTimeMillis)
```

Comment:
I realize this isn't referentially transparent, should be done in `IO`, etc. but I think this is fine for a small test case.  Technically it can fail if you're unlucky and run the test right at the semester switchover time at GS but that also doesn't seem like it is worth sweating over.

Reply:
👍